{{ cookiecutter.project_shortname }}/setup.py

```diff
@@ -10,13 +10,16 @@
 readme = open('README.rst').read()
 history = open('CHANGES.rst').read()
 
+invenio_version = '>=2.0.3'
+
 requirements = [
     'Flask>=0.10.1',
     'six>=1.7.2',
-    'Invenio>=2.0.3',
+    'Invenio{}'.format(invenio_version),
```

Comment:
this should be removed as none of the packages should depend on *Invenio*. There will be *Flask-AppFactory* and *Invenio-Ext(ensions)* packages ...